src/server/graphql/mutations/updateUser.js

```diff
@@ -16,7 +17,23 @@ export default {
       throw new LGNotAuthorizedError()
     }
 
-    await updateUser(values)
+    const user = await getUser(values.id)
+
+    await echoUpdateUser({id: values.id, phaseNumber: values.phaseNumber})
+
+    if (!_identicalRoles(user.roles, values.roles)) {
+      await idmUpdateUser({
```

Comment:
Cleaner to just include this logic into the same `updateUser` server action - that way it can be reused if necessary without duplicating the separate idm call.